scripts/test_deps.jl

```diff
@@ -0,0 +1,20 @@
+function get_deps(pkg, deps = Set{String}())
+    for d in Pkg.dependents(pkg)
+        push!(deps, d)
+        get_deps(d, deps)
```

Comment:
Hm, maybe we should just test direct dependencies? I'm not sure, but this might take a very long time...